src/Compilers/CSharp/Test/Emit/CodeGen/CodeGenAwaitForeachTests.cs

```diff
@@ -4860,9 +4863,52 @@ public sealed class AsyncEnumerator : System.IAsyncDisposable
                 Diagnostic(ErrorCode.WRN_DeprecatedSymbol, "foreach").WithArguments("C.AsyncEnumerator.MoveNextAsync()").WithLocation(7, 15),
                 // (7,15): warning CS0612: 'C.AsyncEnumerator.Current' is obsolete
                 //         await foreach (var i in new C())
-                Diagnostic(ErrorCode.WRN_DeprecatedSymbol, "foreach").WithArguments("C.AsyncEnumerator.Current").WithLocation(7, 15)
-                );
-            // Note: Obsolete on DisposeAsync is not reported since always called through IAsyncDisposable interface
```

Comment:
So the comment was incorrect - we do not go through IAsyncDisposable interface?

Reply:
Yes, this was changed in [PR](https://github.com/dotnet/roslyn/pull/72598) but this comment in test was missed